### ISSUE TITLE
Thoughts About Auto-saving

### ISSUE THREAD

**Issue by Riajyuu** [NickeManarin/ScreenToGif#511]
as per #356 , I have sth in mind about auto-saving feature:

This is the camera UI, I believe.

![image](https://user-images.githubusercontent.com/16912240/62996126-0eb7cb80-be53-11e9-8fce-8496bbadaa47.png)

On the area I drew the square, you can put a radio list looks like:

- Quickly save to clipboard
- Quickly save to directory: [ directory select box here ]

Also, perhaps we need to add another section about quick saving in the app option window as well? Because this is the best place to explain what `quickly save` means: save without editing.

This section may includes: 
- checkbox to decide whether to keep project files
- record technology to use, gifski, WMP or ffmpeg. Give a default value that suits most cases, despite that it requires a check about component existence to disable certain options if failed.

And perhaps more here in this section?

**Comment by MISTikus** [comment]
I think that saving to clipboard would be super-awesome!)
Already waiting this feature)

**Comment by NickeManarin** [comment]
FYI, saving to clipboard does not work the same way as a normal image. Only the path or file (same as `Ctrl+C` the file on Windows Explorer) can be set to the clipboard.

**Comment by MISTikus** [comment]
Yes, i know it)
I meant autosave to clipboard with chosen link (file system or cloud) with some kind of setting...

**Comment by Kingwl** [comment]
It's very helpful that could saving to clipboard. 

We may save the gif into a temporary file?

**Comment by cmalard** [comment]
Would be interested as well is an mode that doesn't involve the editor 😉

I've actually set the option "Start with" to "Screen recorder".
And for every record that I do, I repeat the same actions:
- Record[/Pause]/Stop
- Image -> add Progress -> Apply
- File -> Save as -> Save

Would love to be able to just "Record[/Pause]/Stop" and have all the others done automatically 😉
Maybe with a new option to activate the editor or not after screen capture and some default options like display or not Progress.

Thanks for the already excellent work :beers:

**Comment by ZoserLock** [comment]
Yes, other programs that do not create gif have a good workflow. For example greenshot Print screen key -> Select Area -> Image is saved with a random name in some location and also the image is saved to the clipboard. 

Would be nice something like that for the gif. One button -> Select Area -> Finish -> Gif saved in one location and copied to clipboard.

At the moment I can quick select an area and record a gif, but then there is the tedius part of saving the file in one location and selecting filename, format and stuff.

In any case realy nice app!
